.github/workflows/pr_metrics.yml

```diff
@@ -0,0 +1,59 @@
+name: Post-Sprint PR metrics
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: issue metrics
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: read
+
+    steps:
+      - name: Set REPO_LIST environment variable
+        run: |
+          echo "REPO_LIST=repo:ihmeuw/vivarium repo:ihmeuw/vivarium_public_health repo:ihmeuw/vivarium_inputs repo:ihmeuw/vivarium_cluster_tools repo:ihmeuw/risk_distributions" >> $GITHUB_ENV
+      - name: Get dates for previous sprint
+        shell: bash
+        run: |
+          # Calculate the date two weeks ago
+          fourteen_days_ago=$(date -d "14 days ago" +%Y-%m-%d)
+          
+          # Calculate yesterday's date
+          yesterday=$(date -d "today" +%Y-%m-%d)
+          
+          # Set an environment variable with the date range
+          echo "$fourteen_days_ago..$yesterday"
+          echo "last_two_weeks=$fourteen_days_ago..$yesterday" >> "$GITHUB_ENV"
+
+      - name: Get issue metrics for PRs closed in last sprint
+        uses: github/issue-metrics@v3
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SEARCH_QUERY: $${{ env.REPO_LIST }} is:pr closed:${{ env.last_two_weeks }} -reason:"not planned"'
```

Comment:
what does `-reason:"not planned"` do?

Reply:
i just copied it from the template TBH but looking into it more I think it's proably not appropriate for PRs